components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -27,57 +26,16 @@
 #include "chrome/browser/net/system_network_context_manager.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/ui/browser.h"
-#include "chrome/browser/ui/scoped_tabbed_browser_displayer.h"
 #include "chrome/common/chrome_paths.h"
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "content/public/browser/browser_thread.h"
-#include "extensions/common/url_pattern.h"
-#include "net/base/load_flags.h"
-#include "net/traffic_annotation/network_traffic_annotation.h"
-#include "services/network/public/cpp/resource_request.h"
-#include "services/network/public/cpp/simple_url_loader.h"
-
-// Fetch headers from the referral server once a day.
-const int kFetchReferralHeadersFrequency = 60 * 60 * 24;
-
-// Maximum size of the referral server response in bytes.
-const int kMaxReferralServerResponseSizeBytes = 1024 * 1024;
 
 // Default promo code, used when no promoCode file exists on first
 // run.
 const char kDefaultPromoCode[] = "BRV001";
 
-namespace {
```

Comment:
Need to keep this as it contains the code for initializing the referral and sending the platform identifier for stats reporting.